docs/api-reference/spec/firework-v3-openapi.json

```diff
@@ -7,7 +7,7 @@
     },
     "servers": [
         {
-            "url": "https://api.flare.io/firework/v3"
+            "url": "https://firework-api:5000/firework/v3"
```

Comment:
Thats probably bad?

Can you please generate it from prod instead?

Reply:
Ah my bad, I didn't notice this, sorry.

Reply:
Fixed.